AdminUi/apps/admin_ui/lib/components/shared/date_filter.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:admin_ui/core/constants.dart';
+import 'package:flutter/material.dart';
+
+class DateFilter extends StatefulWidget {
```

Comment:
when no date is selected I see the current date

Reply:
Changes made.